plugin/plugin.go

```diff
@@ -572,6 +581,15 @@ func (p *OrmPlugin) renderGormTag(field *Field) string {
 	if preload != nil {
 		gormRes += fmt.Sprintf("preload:%s;", strconv.FormatBool(*preload))
 	}
+	if clear != nil {
+		gormRes += fmt.Sprintf("clear:%s;", strconv.FormatBool(*clear))
+	}
+	if replace != nil {
+		gormRes += fmt.Sprintf("replace:%s;", strconv.FormatBool(*replace))
+	}
+	if append != nil {
+		gormRes += fmt.Sprintf("append:%s;", strconv.FormatBool(*append))
```

Comment:
I may be missing it, I don't see these as gorm tags here http://v1.gorm.io/docs/models.html#Struct-tags-for-Associations, and aren't these options mutually exclusive, so an `else if` chain might make more sense?

Reply:
Hi @Calebjh, yeah they are not gorm tags but rather instructions for us on how to deal with associations.
I probably didn't need have them in `message GormTag `

Reply:
I have removed from gormTag message and used else if chain.
Please let me know if you think this should be done somewhere else.